app/views/ExposureHistory/History.tsx

```diff
@@ -63,7 +63,9 @@ const History = ({ exposureHistory }: HistoryProps): JSX.Element => {
       <ScrollView style={styles.container} alwaysBounceVertical={false}>
         <View>
           <View style={styles.headerRow}>
-            <Typography style={styles.headerText}>{titleText}</Typography>
+            <View style={{ flexShrink: 1 }}>
```

Comment:
This, combined with the removing the flexWrap in the `headerRow` style allows the expected behavoir for larger language copy / font sizes.